src/commands/deployments/disconnectRepo.ts

```diff
@@ -3,13 +3,20 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { DeploymentsTreeItem } from "vscode-azureappservice";
+import { DeploymentsTreeItem, disconnectRepo as disconnectRepository } from "vscode-azureappservice";
 import { IActionContext } from "vscode-azureextensionui";
+import { SiteTreeItem } from '../../explorer/SiteTreeItem';
 import { ext } from "../../extensionVariables";
+import { localize } from '../../localize';
 
 export async function disconnectRepo(context: IActionContext, node?: DeploymentsTreeItem): Promise<void> {
     if (!node) {
         node = await ext.tree.showTreeItemPicker<DeploymentsTreeItem>(DeploymentsTreeItem.contextValueConnected, { ...context, suppressCreatePick: true });
     }
-    await node.disconnectRepo(context);
+
+    if (node.parent instanceof SiteTreeItem) {
+        await disconnectRepository(context, node.parent.root.client, node.parent.root);
+    } else {
+        throw Error(localize('actionNotSupported', 'Action not supported.'));
```

Comment:
Could you just make the error like this? https://github.com/microsoft/vscode-azureresourcegroups/pull/15/files?diff=split&w=1#diff-7c2b04b667e5fe72febaf888c6ee1a7cR12

I like to try to keep consistent with whatever Eric's doing so our extensions feel like they're from the same company haha.

Reply:
Lol tbf, I should be copying Alex's message since he wrote it first

![1](https://media.giphy.com/media/fCUnd2enoRNfs37vAp/giphy.gif)

Reply:
Lol. I will change it since that error is already in master.